### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Update readme and add missing steps.

I added three things: 
1. cmd command to upload launcher using username and password
2. steps to create env and install dependencies when running from source
3. some helpful links from docs and community

### CODE REVIEW

**Review by tokejepsen — APPROVED** [review]
Resolves #92

File: README.md [hunk 1]
```diff
@@ -54,11 +59,30 @@ AYON can be executed either from live sources (this repository) or from
...
+```
+#### Run
 AYON can be run directly from sources by activating virtual environment:
 
 ```sh
 poetry run python start.py &args
 ```
+> For Windows, you can use this alternatively
+> ```
+> tools/ayon_console.bat
```
Comment (iLLiCiTiT) [inline README.md:83]: I would not suggest using `ayon_console.bat`.
```suggestion
> tools/manage.ps1 run
```

File: README.md [hunk 2]
```diff
@@ -117,3 +141,8 @@ Environment variables that are set for backwards compatibility with openpype add
 
 ## Developer mode
 Developer mode enables to skip standard distribution process and use local sources of addon code. This is useful for development of addon. Developer mode must be enabled and configured on AYON server. To use it in AYON launcher create dev bundle and use `--use-dev` argument, or define bundle name `--bundle <dev bundle name>` in cli arguments.
+
+## Links
+- [Launcher Dev | Ayon Docs](https://ayon.ynput.io/docs/dev_launcher)
+- [Ayon Developer Mode – Guide | Ayon Forums](https://community.ynput.io/t/ayon-developer-mode-guide/993)
+- [How to keep up with Ayon updates? | Ayon Forums](https://community.ynput.io/t/how-to-keep-up-with-ayon-updates/1066)
```
Comment (iLLiCiTiT) [inline README.md:148]:
```suggestion
- [How to keep up with AYON updates? | Ayon Forums](https://community.ynput.io/t/how-to-keep-up-with-ayon-updates/1066)
```

File: README.md [hunk 3]
```diff
@@ -117,3 +141,8 @@ Environment variables that are set for backwards compatibility with openpype add
 
 ## Developer mode
 Developer mode enables to skip standard distribution process and use local sources of addon code. This is useful for development of addon. Developer mode must be enabled and configured on AYON server. To use it in AYON launcher create dev bundle and use `--use-dev` argument, or define bundle name `--bundle <dev bundle name>` in cli arguments.
+
+## Links
+- [Launcher Dev | Ayon Docs](https://ayon.ynput.io/docs/dev_launcher)
+- [Ayon Developer Mode – Guide | Ayon Forums](https://community.ynput.io/t/ayon-developer-mode-guide/993)
```
Comment (iLLiCiTiT) [inline README.md:147]:
```suggestion
- [Ayon Developer Mode – Guide | AYON Forums](https://community.ynput.io/t/ayon-developer-mode-guide/993)
```

File: README.md [hunk 4]
```diff
@@ -54,11 +59,30 @@ AYON can be executed either from live sources (this repository) or from
...
+#### Run
 AYON can be run directly from sources by activating virtual environment:
 
 ```sh
 poetry run python start.py &args
 ```
+> For Windows, you can use this alternatively
+> ```
+> tools/ayon_console.bat
+> ```
```
Comment (iLLiCiTiT) [inline README.md:84]: #### Windows
```
./tools/manage.ps1 run
```

#### Linux & macOS
```
./tools/make.sh run
```

File: README.md [hunk 5]
```diff
@@ -41,9 +41,14 @@ Create installer information from json file on server and upload the installer f
 ### Windows
 Run `./tools/manage.ps1 upload --server <your server> --api-key <your api key>`
 
+*Or,* Run `./tools/manage.ps1 upload --username <your admin username> --password  <your pasword>`
```
Comment (iLLiCiTiT) [inline README.md:44]:
```suggestion
*Or,* run `./tools/manage.ps1 upload --username <your admin username> --password  <your pasword>`
```

File: README.md [hunk 6]
```diff
@@ -41,9 +41,14 @@ Create installer information from json file on server and upload the installer f
...
+
 ### Linux & macOS
 Run `./tools/make.sh upload --server <your server> --api-key <your api key>`
 
+*Or,* Run `./tools/make.sh upload --username <your admin username> --password  <your pasword>`
```
Comment (iLLiCiTiT) [inline README.md:49]:
```suggestion
*Or,* run `./tools/make.sh upload --username <your admin username> --password  <your pasword>`
```

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**